docs/data-sources/repository/local/sbt_repository.md

```diff
@@ -0,0 +1,49 @@
+---
+subcategory: "Local Repositories"
+---
+
+# Artifactory Local Sbt Repository Data Source
+
+Retrieves a local Sbt repository.
+
+## Example Usage
+
+```hcl
+data "artifactory_local_sbt_repository" "terraform-local-test-sbt-repo" {
+  key = "terraform-local-test-sbt-repo"
+}
+```
+
+## Argument Reference
+
+The following argument is supported:
+
+* `key` - (Required) the identity key of the repo.
+
+## Attribute Reference
+
+There are no unique attributes for , along with the [common list of attributes for the local repositories](local.md):
+
+[//]: # (Are these attributes supported for this repo type?)
```

Comment:
?

Reply:
There is a list of attributes here for an sbt repository that is copied and pasted from gradle repository list. I'm not sure if an sbt repository has those attributes though. The data structure that its built on does, but the resource file doesn't list those attributes.

Reply:
Ok. Regardless of what we end up putting in this doc file, what we have here is not correct.